dali_tf_plugin/stub/dali_stub.cc

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/c_api.h"
+
+void daliInitialize() {
```

Comment:
Can we generate this stub from header directly instead of hand writing it?
[This script](https://android.googlesource.com/platform/frameworks/native/+/0469dd6/opengl/libs/GLES_trace/tools/genapi.py) is used by android to generate wrappers for OpenGL calls. Maybe we can reuse it for our purpose?